### PR TITLE
build: prep for 2.12.0 release

### PR DESCRIPTION
**Pull request by macintoshpie**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,74 @@
+# SEED Version 2.12.0
+
+Date Range: 06/22/21 - 09/27/21
+
+Closed Issues and Features (Total: 65):
```
Comment (nllong) [inline CHANGELOG.md:5]: dood, look at the progress! good work y'all!

**Review by nllong — APPROVED** [review]
Looks great. 

Quick question, when I run `npm update` locally it updates the committed lock files, should we do that for this release?

**Comment by macintoshpie** [comment]
> Quick question, when I run npm update locally it updates the committed lock files, should we do that for this release?

Hmm, Im a bit hesitant to run a full npm update that hasn't been tested. Should we just manually update the package version in the shrinkwrap file?

**Comment by nllong** [comment]
> > Quick question, when I run npm update locally it updates the committed lock files, should we do that for this release?
> 
> Hmm, Im a bit hesitant to run a full npm update that hasn't been tested. Should we just manually update the package version in the shrinkwrap file?

Understood. Don't touch it now. We can do this post release!

**Pull request merged**